### DISCUSSION TITLE
Project: 📚 Accessibility features documentation

### DISCUSSION THREAD

**Discussion by thibaudcolas** [wagtail/outreachy#2] [Ideas]
Please use this Discussions thread to ask anything and everything relating to our [Accessibility features documentation](https://github.com/wagtail/outreachy/blob/main/project-ideas.md#accessibility-features-documentation) project idea.

Don’t hesitate to answer other people’s questions and mark this as a contribution on the Outreachy website if you’re particularly happy with your answer.

**Comment by triumphant-fortune** [comment]
Hello everyone please I am having trouble shooting with the markdown table with links to my website I am having difficulty with it do I have to fork and clone to my local machine? need help here, please....

**Reply by thibaudcolas** [reply]
@triumphant-fortune 👋 this question isn’t about that specific project so in the future I’d recommend starting a separate discussion!

Re the Markdown table with links, have you read all the resources under [Checklist: open source contributions](https://github.com/wagtail/outreachy/blob/main/contributor-guide.md#checklist-open-source-contributions)? There’s lots on there about how a fork works. I’d expect it’s all doable in your browser, though knowing how to do it on a local machine is definitely a valuable skill.

For Markdown tables specifically, if you struggle with the formatting, there are tools online that can help check it for issues, or convert from a table in another format, to Markdown.

**Reply by triumphant-fortune** [reply]
@thibaudcolas Thanks so much for the info...
 I will work towards it 👍🏾.

**Comment by ArafatAbdussalam** [comment]
Hi @thibaudcolas  Do you mind checking this PR which I created on how to contribute for new and upcoming participants [pull #48](https://github.com/wagtail/outreachy/pull/48)

**Reply by thibaudcolas** [reply]
@ArafatAbdussalam 👋 this discussion thread is specifically for the [Accessibility features documentation](https://github.com/wagtail/outreachy/blob/main/project-ideas.md#accessibility-features-documentation) project idea. Please keep on topic in the future.

**Comment by waju-ire** [comment]
Hi @thibaudcolas,I have a question concerning the blogs, do our experiences have to be entirely centered around tech/ product building? For example, can a person write about accessibility as he has observed in community building?

**Reply by thibaudcolas** [reply]
@waju-ire if you think there’s an angle, I’d love to read that!

**Comment by thibaudcolas** [comment]
Q: What accessibility features are we talking about?

**Reply by thibaudcolas** [reply]
At this stage we’re considering all features which we’ve identified as part of our in-progress [accessibility audit](https://github.com/wagtail/roadmap/issues/46). In particular all features needed according to the following ATAG requirement:

> [A.4.2.1](https://www.w3.org/TR/2015/REC-ATAG20-20150924/Overview.html#gl_a42) Describe Accessibility Features: For each [authoring tool](https://www.w3.org/TR/ATAG20/#def-Authoring-Tool) feature that is used to meet [Part A](https://www.w3.org/TR/ATAG20/#part_a) of ATAG 2.0, at least one of the following is true: (Level A)
> - (a) Described in the Documentation: Use of the feature is explained in the authoring tool's [documentation](https://www.w3.org/TR/ATAG20/#def-Documentation); […]

At this stage, here are the accessibility features we have identified as needing documentation:

- Image title fields’ usage as alt text
- Page-level keyboard shortcuts
- Skip link
- Collapsible sections
- Link to specific collapsible sections
- Collapse/expand all
- Minimap
- Session time limit
- Editing of headings and elements nesting in rich text fields
- Text search
- Browser-level UI settings
  - Sidebar expanded/collapsed
  - Rich text toolbar pinned/unpinned
  - Minimap opened/closed
  - Currently-open side panel
- Profile-level UI settings
  - Language
  - Time zone
  - Notification settings
  - Admin interface theme
- Live preview
- Command palette available commands

**Reply by codewithmide** [reply]
Hi @thibaudcolas, thanks for sharing this list. Can we start working on them now? Because some of us are done with tasks related to writing articles.

**Reply by meli-imelda** [reply]
Yes you can @codewithmide 
Check the contribution guide to see how to proceed with it

**Reply by cedarworth** [reply]
Thanks @thibaudcolas once again, I worked on Live preview already and reported it [here](https://github.com/wagtail/guide/issues/325)

**Comment by thibaudcolas** [comment]
Q: what kind of accessibility experience would you recommend writing the blog post about?

**Reply by thibaudcolas** [reply]
Any personal experience is great. It doesn’t have to be about web / digital accessibility. I prefer something relating to your personal experience or just past experience if possible. If you have trouble thinking of a topic, here is a [list of accessibility features in Wagtail](https://github.com/wagtail/outreachy/discussions/2#discussioncomment-7248612) which would all be interesting to see covered.

**Comment by thibaudcolas** [comment]
Q: how much overlap is there between accessibility and SEO? Is it a dev’s job or content author?

**Reply by thibaudcolas** [reply]
Quite a bit of overlap – things like descriptive link text with the right keywords, and descriptive alt text, will be impactful for both human users and search engines. In particular, it’s really important for screen reader users _and_ search engines if they can make sense of different pieces of content without having read all the text around.

This is a shared responsibility – content authors write appropriate information, developers make sure the right fields are present in the CMS and the content is used correctly in templates.

**Comment by thibaudcolas** [comment]
Q: do I need to do coding for this project?

**Reply by thibaudcolas** [reply]
No – there is no need to do coding or work with developer tools. If you are interested nonetheless, we recommend:

- Trying out the Gitpod development environment on our [bakerydemo](https://github.com/wagtail/bakerydemo) project
- Preparing the internship application with some room for small coding tasks.

**Comment by ArafatAbdussalam** [comment]
Hi @thibaudcolas I will like to ask if the end users of the accessibility features are web content authors with disabilities or the users we are focused on are the users who eventually become the web content readers of the web page.

**Reply by thibaudcolas** [reply]
@ArafatAbdussalam this is about documentation in Wagtail’s editor guide, so has nothing to do with the site’s audience.

**Reply by ArafatAbdussalam** [reply]
Thanks for the response @thibaudcolas . I did more findings to understand the project better.

**Comment by ArafatAbdussalam** [comment]
> @ArafatAbdussalam 👋 this discussion thread is specifically for the [Accessibility features documentation](https://github.com/wagtail/outreachy/blob/main/project-ideas.md#accessibility-features-documentation) project idea. Please keep on topic in the future.

I apologize for not having followed the discussion rules.

**Comment by nouman17610** [comment]
When will we get editor access to the Wagtail website and guide.wagtail.org for the tasks mentioned below?

- create a new page demonstrating some of Wagtail’s accessibility features.
- identify a feature of Wagtail that isn’t documented, and report it back on the(https://github.com/wagtail/guide/issues). Make sure to
- explain what the feature is, why, and how you think it should be documented.
-  Draft the new content to document the feature of Wagtail which you’ve reported isn’t documented. Add it to the issue you created as a new comment so that we can provide feedback.
-   make the necessary changes in the CMS to add the feature documentation.

**Reply by Ranecodes** [reply]
Yes please, was wondering the same thing. Has anyone gotten editor access at all?

**Reply by thibaudcolas** [reply]
You will get editor access once you’ve done all the previous tasks, and have asked me, and I have replied. Simple as that.

**Reply by nouman17610** [reply]
Okay, thank you. I have reached that level and sent you an email too.

**Reply by Ozyugoo** [reply]
I have also sent you an email too, could you please check? Thank you

**Comment by ArafatAbdussalam** [comment]
Hi @thibaudcolas I will like to ask questions on the grade 7 requirement on Hemming way. Does the grade affect articles on accessibility and technical writing or does it just apply to what we learnt personally?

**Reply by ginandze** [reply]
I think it applies to all the blog posts you write.

**Reply by ArafatAbdussalam** [reply]
Thank you for the response @ginandze

**Reply by ginandze** [reply]
Glad to be of help :)

**Comment by codewithmide** [comment]
Hello @thibaudcolas, I hope this finds you well. I would like to gain editor access to the Wagtail site, I am done with other tasks.

**Reply by ginandze** [reply]
Hello @thibaudcolas, I too would like to be granted access to the Wagtail site. Patiently waiting.

**Reply by meli-imelda** [reply]
Try getting to him on slack or via email as well

**Comment by Amalearner** [comment]
Hello @thibaudcolas trust you are having a good day.  I'd like to have access as an editor to the wagtail website.

**Comment by Stanzin7** [comment]
hello @thibaudcolas, thanks for giving pointers to the project. This is my medium post on my summer work at Drupal CMS. https://shorturl.at/wJRTY. Can you please grant me an access to the editor of the wagtail website. Thanks )

**Comment by cedarworth** [comment]
@thibaudcolas , I raised a couple of issues at [guide project issue tracker](https://github.com/wagtail/guide/issues)  as follows
[issue1](https://github.com/wagtail/guide/issues/325)  and   
[issue2](https://github.com/wagtail/guide/issues/327)

**Comment by nouman17610** [comment]
Hello @everyone guys anyone got editorial access to Wagtail Guide yet?
